Userland/Services/WindowServer/Menu.cpp

```diff
@@ -501,16 +501,19 @@ void Menu::start_activation_animation(MenuItem& item)
     auto window = Window::construct(*this, WindowType::Menu);
     window->set_frameless(true);
     window->set_hit_testing_enabled(false);
-    window->set_opacity(0.8f); // start out transparent so we don't have to recompute occlusions
```

Comment:
Reviewers: Pay special attention to how I refactored `WindowServer::Menu::start_activation_animation(). I'm sure it could be done better somehow.

Reply:
Frankly this animation should be redone to take advantage of the new `Animation` class and use overlays instead of a window. But that's beyond the scope of this PR.

Reply:
I'd like to give that a shot in a subsequent PR! This function did look sort of ad-hoc, so I figured there may be better ways to do it these days.